playbooks/roles/cert-manager/tasks/main.yml

```diff
@@ -1,27 +1,18 @@
 ---
-
-- name: Add the `deadsnakes` PPA for getting Python3.6 on Ubuntu 16.04
-  apt_repository:
-    repo: 'ppa:deadsnakes/ppa'
-
-- name: Download Python3.6
+- name: Install pip
   apt:
-    name: python3.6
-
-- name: Download pip3.6
-  shell:
-    cmd: curl https://bootstrap.pypa.io/get-pip.py | sudo -H python3.6
+    name: python3-pip
 
 - name: Install pipenv to later install python dependencies and virtual environments.
   pip:
     name: pipenv
-    executable: pip3.6
   become: true
 
 - name: Download cert-manager
   git:
     repo: "https://github.com/open-craft/cert-manager"
     dest: "{{ cert_manager_path }}"
+    version: "keith/cert-manager-upgrade"
```

Comment:
@keithgg, I've followed test steps, but noticed that new directories in `/etc/letsencrypt/live/` have `root` group unlike the existing ones. Is that expected?

Reply:
@0x29a Nice find! Yes that's correct. The `ubuntu` group is from my backup. I've rectified it now.